lua/telescope/pickers/entry_display.lua

```diff
@@ -43,7 +43,7 @@ end
 
 local function truncate(str, len)
   -- TODO: This doesn't handle multi byte chars...
-  if vim.fn.strdisplaywidth(str) > len then
+  if #str > len then
```

Comment:
Can can no longer do `strdisplaywidth` because we are not allowed to call `vimL function in a lua loop callback`, but `#str` is not the same. Is there another lua solution for this

Reply:
@tjdevries

Reply:
vim.api.nvim_strwidth() ? I think that will work.

Reply:
@tjdevries nope that didn't cut it for the lua loop. ... hmmmm